static/templates/new_stream_user.hbs

```diff
@@ -1,5 +1,5 @@
 <label class="checkbox add-user-label" id="user_checkbox_{{user_id}}">
     <input type="checkbox" name="user" {{#if checked}}checked="checked"{{#if disabled}} disabled="disabled"{{/if}}{{/if}} data-user-id="{{user_id}}"/>
     <span></span>
-    {{full_name}} ({{email}})
+    {{full_name}} {{#if show_email}}({{email}}){{else}}({{t 'User ID' }}: {{user_id}}; <i>{{t 'email hidden'}}</i>){{/if}}
```

Comment:
I think `em` tags are preferred over `i` tags for generating italics in modern HTML, though it's a bit weird semantically because we use italics for lower-emphasis.  

From an i18n perspective, I think it'd be better to have a single that has both variables `{{t 'User ID: {user_id}; <em>email hidden</em>' }}`, because in some other languages, one might want to adjust the punctuation or word order.

Reply:
Fixed this using `{{tr}}`.